client-next/src/components/MapView/MapView.tsx

```diff
@@ -53,14 +70,33 @@ export function MapView({
         {tripQueryResult?.trip.tripPatterns.length && (
           <LegLines tripPattern={tripQueryResult.trip.tripPatterns[selectedTripPatternIndex] as TripPattern} />
         )}
-        {showPopup && (
+        {showContextPopup && (
           <ContextMenuPopup
             tripQueryVariables={tripQueryVariables}
             setTripQueryVariables={setTripQueryVariables}
-            coordinates={showPopup}
-            onClose={() => setShowPopup(null)}
+            coordinates={showContextPopup}
+            onClose={() => setShowContextPopup(null)}
           />
         )}
+        {showPropsPopup?.feature?.properties && (
+          <Popup
```

Comment:
I got a little sidetracked by the server/client discussion and forgot to review the rest of the code 😬 

Could you please extract this popup into a component, similar to `ContextMenuPopup`?

Reply:
Done in c5dc28984a83a968660c2478c2a6b3301fa7a6d4